### DISCUSSION TITLE
Can I put EssentialsX and other affiliate plugins into my server and publish it?

### DISCUSSION THREAD

**Discussion by 1233Ao** [EssentialsX/Essentials#5139] [Q&A]
Can I put EssentialsX and other affiliate plugins into my server and publish it?

**Comment by pop4959** [comment]
No, the plugin files should not be redistributed directly. Share the [download page](https://essentialsx.net/downloads.html) instead to download the latest version.

**Comment by 1233Ao** [comment]
Do I have to let users download them one by one on the download page, and can I let them run BAT to download plugins from the download page of EssentialsX
